code/js1assignment.js

```diff
@@ -0,0 +1,25 @@
+// String operations
+const myName = "Matilda"
+console.log(myName.length);
+console.log(`Hello, my name is ${myName}`);
+const upperCaseName = myName.toUpperCase()
+console.log(`When my friends see me they shout ${upperCaseName}!`); 
+// or
+console.log(`When my friends see me they shout ${myName.toUpperCase()}!`);
+
+// Numbers operation
+const num = 2
+console.log((num * 2 + 8) / 2 - num);
+
+// Another way to do the above operation
+const result1 = (num * 2 + 8) / 2 - num   //PEMDAS
+console.log(result1);
+
+// Area of circle (22/7 R**2)
+const radius = 7
+const result2 = Math.PI * radius ** 2
+console.log(result2.toFixed(4));
+
+// or 
+const result3 = 22 / 7 * (radius ** 2)    //PEMDAS  --- how come without the brackets it still gives the same result yet multiplication should happen before division? ie. 7 * radius ** 2, then divide the result by 22
+console.log(result3.toFixed(4));
```

Comment:
if you didn't use the parentheses, like 22 / 7 * radius ** 2, it would still be evaluated in the same order because exponentiation has higher precedence than multiplication and division, and multiplication and division are evaluated from left to right. Therefore, the parentheses in this specific case are redundant but help make the code clearer @njaumatilda

Reply:
Sorry... I actually put the brackets in the wrong place... What I meant to indicate was (22 / 7) * radius ** 2 ... But still even without these brackets it gave the correct answer, so my question was how come without the brackets it still gives the same result yet multiplication should happen before division? ie. 7 * radius ** 2, then divide the result by 22